modules/metering-install-prerequisites.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adc
+
+[id="metering-install-prerequisites_{context}"]
+= Prerequisites
+
+Operator Metering requires the following components:
+
+* A StorageClass for dynamic volume provisioning. Metering supports a number of different storage solutions.
+* A Prometheus installation within the cluster configured to do Kubernetes cluster-monitoring. {product-title} 4 clusters have Prometheus enabled by default on installation.
+* 4GB Memory and 4 CPU Cores available cluster capacity.
```

Comment:
We may want to expand on this and go further to say "and at least one node with 2 CPU cores and 2GB memory capacity available" or something which is the most any single pod we have requests at a minimum (the Presto pod).